zcash_client_backend/src/fees/common.rs

```diff
@@ -533,10 +538,10 @@ where
                         let fee_with_dust = (total_in - total_out)
                             .expect("we already checked for sufficient funds");
                         // We can add a change output if necessary.
```

Comment:
At this point we already checked that there are sufficient funds with a change output, and we're removing the assertion this comment is attached to (I can't do it in one suggestion). So this comment is confusing and redundant.